### DISCUSSION TITLE
Rename radiostation added by link.

### DISCUSSION THREAD

**Discussion by melwinek** [music-assistant/support#1934] [Feature requests and ideas]
Changing the name of an element added via a URL link would be very useful. Also ability to assign icon URL. I tried changing it manually in the SQLite database. It was fine for some time, but at some point the names reset to the file names in the URL address.
![image](https://github.com/music-assistant/hass-music-assistant/assets/10649327/02b227dc-1156-444f-83c4-15f43369ddb2)

**Comment by OzGav** [comment]
Please see if this has been fixed for you in b137

**Reply by marcelveldt** [reply]
You cant rename a manually added radio station - for now just re-add it.
Also you shouldnt be manually hacking in the database - that is asking for trouble - please don't.

Functionality to edit metadata will come soon. It might not end up in the 2.0 release but then it will arrive shortly after.

**Comment by OzGav** [comment]
Going through old posts. This has been implemented back in MA 2.9
